lib/node-util.js

```diff
@@ -0,0 +1,22 @@
+// a stand-in for util that does nothing but prevent the existing
+// node:util code paths from exploding
+function nullUtilInspect() {
+  return "";
+}
```

Comment:
in non-node environment (i.e.; a browser) there is no equivalent for `util.inspect` which is why i initially opted for the null implementation.  but since the `MemoryCookieStore` does expose a public `inspect` function then i agree we should try to provide an equivalent output when the fallback is being used so internal usage remains consistent.  

since the memory store is the only point where `util.inspect(...)` is invoked (internally) then i've opted for adding the implementation for this to the fallback `inspect` directly.  this should end up producing the same output as before.